transport/internet/splithttp/h1_conn.go

```diff
@@ -0,0 +1,21 @@
+package splithttp
+
+import (
+	"bufio"
+	"net"
+)
+
+type H1Conn struct {
+	UnreadedResponsesCount int
+	// To reuse response reader, so we won't lose data
+	// If some EOF will accure
```

Comment:
the reason you want to reuse the buf reader is because it sometimes reads more data than it returns, and returns the over-read data on the next `Read` call. so it's not related to EOF